### PR TITLE
implemented an obsidian loader

### PR DESCRIPTION
**Pull request by samahrens**
Issue: #61 

A video demo of it is in the discord

### CODE REVIEW

File: backend/loaders/obsidian_loader.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import os
+from fastapi import UploadFile
+from langchain.document_loaders import ObsidianLoader
```
Comment (grumpyp) [inline backend/loaders/obsidian_loader.py:3]: would it be possible to be independent of LangChain? As all the other stuff is basically implemented without usage of LangChain.

Comment (samahrens) [reply backend/loaders/obsidian_loader.py:3]: I could try, but is there a reason for being independent of them? Their Obsidian loader is more in depth and will probably work better

Comment (grumpyp) [reply backend/loaders/obsidian_loader.py:3]: I am a fan of no dependencies, I can name one example:

In the beginning of AIxplora I used LangChain in combination with Chroma. The bug was so in one of these modules so I had first to dive into it, then I'd need to make a fix as there was no workaround. In the end I decided to make it myself without dependencies.

I would be very happy if you could try to make it without LangChain but would understand if the worload is too big at the moment. 

Then we could create a follow up ticket. What do you think?

Thanks for your contribution once again!

Comment (samahrens) [reply backend/loaders/obsidian_loader.py:3]: Yeah sure, I'll work on doing it without LangChain. What do you think about using similar logic that they used, but in our own function? Their obsidian loader parsed files with different formats pretty well and I think it'd be nice to incorporate that.

**Review by grumpyp — CHANGES_REQUESTED** [review]

**Comment by grumpyp** [comment]
awesome work, thank you!

**Pull request merged**